### PR TITLE
#165273467 integrate coveralls

### PR DESCRIPTION
**Pull request by sanya-kenneth**
### What does this PR do?
- Integrate coveralls code coverage service with ah-backend-invictus repository
- Add coveralls.yml
- Add repo token generated by coveralls service to the coveralls.yml file
- Add coveralls badge to the readme.md file

### What are the relevant user stories?
 [#165273467](https://www.pivotaltracker.com/story/show/165273467)

@engjames @mwinel @archibishop @joelmugaya @3Nakajugo @KabohaJeanMark

### CODE REVIEW

**Review by mwinel — APPROVED** [review]
Good work @sanya-kenneth.

**Review by 3Nakajugo — APPROVED** [review]
Great work @sanya-kenneth

**Review by KabohaJeanMark — APPROVED** [review]
Thank you for the work on integrating coveralls.
Good work following the conventions. 
A small reminder and recommendation. At the end of your PR please add 
[Starts #165273459] when dealing with chores

**Comment by archibishop** [comment]
@sanya-kenneth Coverage badge is showing unknown. Should be showing something like zero. Make the badge point to your branch and see if it will show coverage %.

**Comment by sanya-kenneth** [comment]
@archibishop Let me fix that right away

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+<<<<<<< HEAD
```
Comment (archibishop) [inline README.md]: @sanya-kenneth  It seems changes were not applied when we were doing the rebase. Remove from line 2 - 5 plus line 8 and leave the rest. Then amend the commit and everything will be fine.

**Comment by archibishop** [comment]
@sanya-kenneth Add the necessary lines of code to travis to push the latest coverage to the coveralls

**Comment by engjames** [comment]
Good work @sanya-kenneth but add a newline in .coveralls.yml at the end of the repo_token

File: .coveragerc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[report]
```
Comment (archibishop) [inline .coveragerc]: @sanya-kenneth Remove this file

**Pull request merged**